model/legacy-private/pom.xml

```diff
@@ -32,6 +32,11 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-model-legacy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-jaxrs</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
I received a complaint in the IDE about the LegacyExportImportManager that is calling ValidationUtil.validateClient (which is in server-spi-private) might throw a BadRequestException exception, which is in that package. Maybe we can discuss tomorrow how to handle this. If something needs to move, it would be LegacyExportImportManager + LegacyMigrationManager.

```
/home/aschwart/redhat/workspace/keycloak-pr-1/model/legacy-private/src/main/java/org/keycloak/storage/datastore/LegacyExportImportManager.java:437:42
java: cannot access javax.ws.rs.BadRequestException
  class file for javax.ws.rs.BadRequestException not found
```

Reply:
Thank you, let's keep this as is, even though this is not ideal. Perhaps a new exception would be better than this.

Could you please add a comment to this place in `pom.xml` referring this comment thread?